examples/README.md

```diff
@@ -38,3 +38,32 @@ and build this example Level Zero development package should be installed.
 * Level Zero headers and libraries
 * compatible GPU with installed driver
 * set UMF_BUILD_GPU_EXAMPLES, UMF_BUILD_LIBUMF_POOL_DISJOINT, UMF_BUILD_LEVEL_ZERO_PROVIDER and UMF_ENABLE_POOL_TRACKING CMake configuration flags to ON
+
+## IPC example with shared memory
+This example also demonstrates how to use UMF IPC API. The example creates two
+processes: a producer and a consumer that communicate in the following way
+(the initial value 140722582213392 in the shared memory is quasi-random):
+- Consumer starts
+- Consumer creates a socket
+- Consumer listens for incoming connections
+- Producer starts
+- Producer's shared memory contains a number: 140722582213392
+- Producer gets the IPC handle
+- Producer creates a socket
+- Producer connects to the consumer
+- Consumer connects at IP 127.0.0.1 and port 47770 to the producer
+- Producer sents the IPC handle to the consumer (24 bytes)
```

Comment:
`(24 bytes)` is implementation details that might be changed. Let's omit such details.

Reply:
Done